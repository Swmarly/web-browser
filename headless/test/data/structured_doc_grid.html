<!DOCTYPE html>
<html lang="en">
<body>
<div role="grid">
  <div role="row">
    <span role="columnheader">Header 1</span>
    <span role="columnheader">Header 2</span>
  </div>
  <div role="row">
    <span role="gridcell">Cell 1</span>
    <span role="gridcell">Cell 2</span>
  </div>
</div>
</body>
</html>
