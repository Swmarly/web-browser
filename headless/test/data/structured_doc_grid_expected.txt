{
   "lang": "en",
   "type": "Document",
   "~children": [ {
      "type": "Table",
      "~children": [ {
         "type": "TR",
         "~children": [ {
            "attributes": [ {
               "O": "Table",
               "Scope": "Column"
            }, {
               "O": "Table",
               "RowSpan": 1.0
            }, {
               "ColSpan": 1.0,
               "O": "Table"
            } ],
            "type": "TH",
            "~children": [ {
               "type": "NonStruct"
            } ]
         }, {
            "attributes": [ {
               "O": "Table",
               "Scope": "Column"
            }, {
               "O": "Table",
               "RowSpan": 1.0
            }, {
               "ColSpan": 1.0,
               "O": "Table"
            } ],
            "type": "TH",
            "~children": [ {
               "type": "NonStruct"
            } ]
         } ]
      }, {
         "type": "TR",
         "~children": [ {
            "attributes": [ {
               "Headers": [ "TH" ],
               "O": "Table"
            }, {
               "O": "Table",
               "RowSpan": 1.0
            }, {
               "ColSpan": 1.0,
               "O": "Table"
            } ],
            "type": "TD",
            "~children": [ {
               "type": "NonStruct"
            } ]
         }, {
            "attributes": [ {
               "Headers": [ "TH" ],
               "O": "Table"
            }, {
               "O": "Table",
               "RowSpan": 1.0
            }, {
               "ColSpan": 1.0,
               "O": "Table"
            } ],
            "type": "TD",
            "~children": [ {
               "type": "NonStruct"
            } ]
         } ]
      } ]
   } ]
}
